M/MFEM/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MFEM_seq"
+version = v"4.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mfem/mfem.git", "a1f6902ed72552f3e680d1489f1aa6ade2e0d3b2")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mfem*
+mkdir build; cd build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS=1 \
+    ..
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+platforms = expand_cxxstring_abis(platforms)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libmfem", :libmfem)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"5")
```

Comment:
Looks like GCC 5 doesn't like boolean enums, but GCC 6 does:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"6")
```

```
[10:40:12] /workspace/srcdir/mfem/fem/restriction.cpp: In member function ‘void mfem::NCL2FaceRestriction::ComputeScatterIndicesAndOffsets(mfem::ElementDofOrdering, mfem::FaceType)’:
[10:40:12] /workspace/srcdir/mfem/fem/restriction.cpp:2108:66: error: could not convert ‘(mfem::FaceType)type’ from ‘mfem::FaceType’ to ‘bool’
[10:40:12]                 (type==FaceType::Interior? "interior" : "boundary") <<
[10:40:12]                                                                   ^
```